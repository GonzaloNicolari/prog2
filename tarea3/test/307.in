# prueba de esAVL
es_AVL
insertar_en_binario (1,ab)
insertar_en_binario (100,abc)
es_AVL
insertar_en_binario (10,aba)
es_AVL
insertar_en_binario (1,abb)
es_AVL
insertar_en_binario (1,abd)
es_AVL
insertar_en_binario (1,aaaaa)
# los dos subarboles son no vacios y AVL. Pero el árbol no es AVL
es_AVL
insertar_en_binario (1,aaaab)
insertar_en_binario (1,aaaaz)
# las alturas de los subárboles son iguales. Pero el árbol no es AVL.
es_AVL
insertar_en_binario (1,a)
es_AVL
Fin
