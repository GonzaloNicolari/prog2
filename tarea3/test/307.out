1>#  prueba de esAVL.
2>AVL.
3>Insertado.
4>Insertado.
5>AVL.
6>Insertado.
7>No AVL.
8>Insertado.
9>No AVL.
10>Insertado.
11>No AVL.
12>Insertado.
13>#  los dos subarboles son no vacios y AVL. Pero el árbol no es AVL.
14>No AVL.
15>Insertado.
16>Insertado.
17>#  las alturas de los subárboles son iguales. Pero el árbol no es AVL..
18>No AVL.
19>Insertado.
20>AVL.
21>Fin.
